src/content/docs/new-relic-one/use-new-relic-one/ui-data/automaps.mdx

```diff
@@ -0,0 +1,195 @@
+
+<figcaption>
+Here's an example of where you could start troubleshooting with automap.
+</figcaption>
+
+## How it works [#how-works]
+
+New Relic automap displays relationships between entities in your architecture. For version 1, the relationships displayed in automap are based on direct calls between services. For example, APM services are represented as vertices (hexagons) and direct calls are represented as edges (lines) on the graph. This shows you how these services depend on one another. 
+
+ Automap displays only the entities related to the source entity that have experienced a change in health status within three hours after the automap was launched. So, it may not show all entities related to the source entity. To understand entities and relationships, as well as how the entity platform synthesizes both from telemetry data, see [entities](https://docs.newrelic.com/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic/) and [relationships](https://docs.newrelic.com/docs/apis/nerdgraph/examples/nerdgraph-relationships-api-tutorial/).
+
+<Callout variant="important">
+Automap auto-expand logic relies on the health status of an entity, which is largely determined by the alert status of that entity. For optimal troubleshooting with automap, ensure that your [alerts are properly configured](docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/).
+</Callout>
+
+## Open automap [#open-automap]
+
+Automap appears on any **Related entities** pane for supported entity types:
+
+  1. In **Explorer**, insert a filter to find your entity/service (for example, `alerts`).
+  2. Click on the entity/service.
+  3. On the **Summary page**, go to **Related entities** in the right pane.
+  4. Click **Map view**: 
+  
+    ![Screenshot showing the Map view button](./images/map-view-button.png "Screenshot showing the Map view button")
+
+The **Map view** button is only available for the following types of entities:
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "200px" }}>Entity type
+        
+      </th>
+      <th>Description</th>
+      <th>Icon</th>
+    </tr>
+  </thead>
+    <tbody>
+    <tr>
+      <td>
+        APM|APPLICATION
+      </td>
+      <td>
+        Application service instrumented with a New Relic APM agent
+      </td>
+      <td>
+        ![Screenshot of application icon](./images/application-icon.png "Screenshot of application icon")
+      </td>
+      <td>
+        
+      </td>
+    </tr>
+    <tr>
+      <td>
+        BROWSER|APPLICATION
+      </td>
+      <td>Browser application instrumented with New Relic Browser agent</td>
+      <td>
+        ![Screenshot of browser application icon](./images/browser-application-icon.png "Screenshot of browser application icon")
+      </td>
+    </tr>
+    <tr>
+      <td>
+        EXT|SERVICE
+      </td>
+      <td>
+        Services instrumented with OpenTelemetry, including Pixie-instrumented services.
```

Comment:
some of the sentences in the table have periods, others don't

Reply:
@stumblefiend, I removed the period!